config.default.yaml

```diff
@@ -415,6 +415,17 @@ solving:
     options: gurobi-default
 
   solver_options:
+    highs-default:
```

Comment:
```suggestion
    highs-default:
      # refer to https://ergo-code.github.io/HiGHS/options/definitions.html#solver
```

Reply:
I found the link to the documentation of the solver options incredibly helpful, should we add them directly to the config?